components/ModulePane/ModuleVersionInfo.tsx

```diff
@@ -72,9 +73,14 @@ export function ModuleVersionInfo({
           : simplur`${-patchDiff} patch version[|s] ahead of`;
     }
 
+    const latestLink = (
+      <QueryLink query={`${module.packument.name}@${latestVersion}`}>
```

Comment:
The version isn't needed

```suggestion
      <QueryLink query={module.packument.name}>
```